api/test/integration/test_dist_analytics.py

```diff
@@ -206,22 +219,199 @@ def test_returns_200_Ok_response_code(self):
         assert response.status_code == 200
 
 
-def test_gadm_dist_analytics_no_intersection():
-    delete_resource_files("9c4b9bb5-0ecd-580d-85c9-d9a112a69b59")
+class TestDistAnalyticsPostWithMultipleAdminAOIs:
+    @pytest_asyncio.fixture(autouse=True)
```

Comment:
I see you explicitly naming the setup fixture for the tests below (in order to get the request and client objects). So could we make this not autouse=True? I find lots of autouse fixtures slows things down and can lead to confusing problems, so try not to use them.

Reply:
Sure, I actually just copied and pasted this and forgot what autouse actually does...

Reply:
Adding the `LifespanManager` context definitely made the integration setup messier. I have another idea for the integration tests and how it can be made more sustainable.

Reply:
Will wait on this, I think we're going to change the tests based on Gary's refactoring